docs/network protocol.md

```diff
@@ -31,6 +31,9 @@ There are also a number of community-supported libraries available that implemen
 | GameMaker: Studio 2.x+        | [see Discord](https://discord.com/channels/731205301247803413/1166418532519653396)                 |                                                                                 |
 
 ## Synchronizing Items
+After a client connects, it will receive all previously collected items for its associated slot in a [ReceivedItems](#ReceivedItems) packet. This will include items the client may have already processed in a previous play session.  
+To ensure the client is able to reject those items if it needs to, each item in the packet will have an associated `index` argument. You will need to find a way to save the "last processed item index" to the player's local savegame, a local file, or something to that effect. Before connecting, you should load that "last processed item index" value and compare against it in your received items handling.
```

Comment:
I think an example of why a client might need to do this would be helpful. Also a tense fix. Also worried this makes it more complicated though
```suggestion
To ensure the client is able to reject those items if it needs to, such as ignoring already-received items, each item in the packet has an associated `index` argument. You will need to find a way to save the "last processed item index" to the player's local savegame, a local file, or something to that effect. Before connecting, you should load that "last processed item index" value and compare against it in your received items handling.
```

Reply:
I have some faith here that the person looking at this will usually be working on or theorising a world implementation, such that when they read the previous sentence that says `This will include items the client may have already processed in the previous play session` that will evoke in them the thought "Oh, I need to do something about that".

Maybe you're right though and it should be even more obvious. But I think the sentence *is* getting a bit complicated at that point

Reply:
Yeah, I worry about that too. It's probably fine as is (just do "will have" -> "has"). And if people are confused, maybe that'll prompt them to think through it, or, if not, to ask something in the server